types/2022-08-01/Checkout/Sessions.d.ts

```diff
@@ -3146,7 +3151,7 @@ declare module 'stripe' {
           description?: string;
 
           /**
-           * A list of items, each with an attached plan, that the customer is subscribing to. Prefer using `line_items`.
+           * This parameter is deprecated. Use the line_items parameter on the Session instead.
```

Comment:
This deprecation was mentioned in other changelogs, but not in the one for Node. Is that intentional?